### PR TITLE
doc: add installation guide #85

### PR DESCRIPTION
**Pull request by Wence-May**
## Purpose
add an installation guide (with common browsers: Edge, Chrome, 360 Apeed & 360 Safety)

## Proposed changes
INSTALLTION.md is added to the root directory.

#### Open Questions and Pre-Merge TODOs
- [x] issue #85 

#### Types of changes
_What types of changes does your code introduce to hypertrons-crx?_
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Installation Guide
+
+We welcome users to install and try our Hpypertrons Chrome Extension. If you have any feedback, please post it in the issue area (with description of the problem and browser configurations). We will actively respond and solve it!
```
Comment (heming6666) [inline INSTALLATION.md:3]: `Hpypertrons` should be `Hypertrons` here.

**Review by heming6666 — CHANGES_REQUESTED** [review]
Great job! Thanks @Wence-May .

File: INSTALLATION.md (hunk 1 above)
Comment (LiuChangFreeman) [reply INSTALLATION.md:3]: Now we have provided the `hypertrons.zip`, so you can update the procedure `unzip hypertrons.crx`.

Comment (Wence-May) [reply INSTALLATION.md]: All right! problems fixed.

**Review by heming6666 — APPROVED** [review]

**Pull request merged**